cli/package.json

```diff
@@ -2,17 +2,25 @@
   "name": "stoat",
   "version": "0.0.8",
   "description": "Stoat CLI",
-  "main": "src/index.ts",
+  "main": "dist/index.js",
+  "files": [
+    "dist/**/*",
+    "public/**/*"
+  ],
+  "directories": {
+    "lib": "src",
+    "test": "__tests__"
+  },
   "bin": {
     "stoat": "./dist/index.js"
   },
   "scripts": {
     "start": "nodemon --watch 'src/**/*.ts' --exec 'ts-node' src/index.ts",
-    "format": "prettier --write '**/*.{ts,json}'",
-    "format-check": "prettier --check '**/*.{ts,json}'",
+    "format": "prettier --write '**/*.{ts,js,json}'",
+    "format-check": "prettier --check '**/*.{ts,js,json}'",
     "lint": "eslint src/**/*.ts",
     "create": "npm run build && npm run test",
-    "build": "ncc build --source-map --license licenses.txt",
+    "build": "ncc build src/index.ts --source-map --license licenses.txt",
```

Comment:
Because the `main` is a file in the `dist`, which does not exist before the `build` is run, we need to manually feed in the actual `main` file to build.